testify/test_rerunner.py

```diff
@@ -1,11 +1,14 @@
 from __future__ import absolute_import
 from itertools import groupby
 import sys
+from collections import namedtuple
 
 from . import test_discovery
 from .test_runner import TestRunner
 
 
+Test = namedtuple('Test', 'module cls, method')
```

Comment:
I had thought that the string form was the passthru path, but I was wrong:
https://github.com/python/cpython/blob/2.7/Lib/collections.py#L329